docs/resources/vpc_firewall_rules.md

```diff
@@ -6,9 +6,71 @@ description: |-
   This resource manages VPC firewall rules.
   !> Firewall rules defined by this resource are considered exhaustive and will
   overwrite any other firewall rules for the VPC once applied.
-  !> Setting the rules attribute to [] will delete all firewall rules for the
+  !> Setting the rules attribute to {} will delete all firewall rules for the
   VPC which may cause undesired network traffic. Please double check the firewall
   rules when updating this resource.
+  Migrating rules
```

Comment:
Don't have to do this now but we should probably stop using the large description for this YAML front matter.

Reply:
Agreed. I'd like to take a look at this in a future change.